modules/light-clients/08-wasm/testing/mock_engine.go

```diff
@@ -0,0 +1,65 @@
+package wasmtesting
+
+import (
+	wasmvm "github.com/CosmWasm/wasmvm"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+)
+
+const DefaultGasUsed = uint64(1)
+
+var _ types.WasmEngine = &MockWasmEngine{}
```

Comment:
I think we favour this other way of writing the compiler assertion, right?

```suggestion
var _ types.WasmEngine = (* MockWasmEngine)(nil)
```